PublicationsRouter/v2/sword-out/pubrouter-xml/pubrouter.xsd

```diff
@@ -0,0 +1,285 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified"
+    targetNamespace="http://pubrouter.jisc.ac.uk/rioxxplus"
+    xmlns:rioxxterms="http://www.rioxx.net/schema/v2.0/rioxxterms/"
+    xmlns:ali="http://ali.niso.org/2014/ali/1.0"
+    xmlns:dcterms="http://purl.org/dc/terms/"
+    xmlns:vc="http://www.w3.org/2007/XMLSchema-versioning" vc:minVersion="1.1"
+    xmlns="http://pubrouter.jisc.ac.uk/rioxxplus">
+    <xs:import namespace="http://www.rioxx.net/schema/v2.0/rioxxterms/"
+        schemaLocation="http://rioxx.net/schema/v2.0/rioxx/rioxxterms.xsd"/>
+    <xs:import namespace="http://ali.niso.org/2014/ali/1.0"
+        schemaLocation="http://rioxx.net/schema/v2.0/rioxx/ali.xsd"/>
+    <xs:import namespace="http://purl.org/dc/terms/"
+        schemaLocation="http://www.dublincore.org/schemas/xmls/qdc/dcterms.xsd"/>
+    
+    <xs:complexType name="contributor_type">
+        <xs:all>
+            <xs:element name="type" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Type of the contributor, such as "editor".
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="id" minOccurs="0" maxOccurs="unbounded">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Identifier for this contributor, such as ORCID.
+                    </xs:documentation>
+                </xs:annotation>
+                <xs:complexType>
+                    <xs:simpleContent>
+                        <xs:extension base="xs:string">
+                            <xs:attribute name="type" type="xs:string" use="required"/>
+                        </xs:extension>
+                    </xs:simpleContent>
+                </xs:complexType>
+            </xs:element>
+            <xs:element name="email" minOccurs="0" maxOccurs="unbounded" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Email address for this contributor.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="surname" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Surname of this contributor.
+                    </xs:documentation>
+                </xs:annotation>
+            </xs:element>
+            <xs:element name="firstnames" minOccurs="0" type="xs:string">
+                <xs:annotation>
+                    <xs:documentation xml:lang="en">
+                        Comma separated first names for this contributor.
```

Comment:
So space separated? Not sure what you're suggesting here - this is not a repeating element.